CHANGES.md

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+### 1.41 - 2017-01-02
+
+* Added `clippingPlanes` property to `ModelGraphics` and `Cesium3DTileset`, which specifies a `ClippingPlanesCollection` to clip the object. [#5913](https://github.com/AnalyticalGraphicsInc/cesium/pull/5913)
+* Added `terrainClippingPlanes` property to `Globe` which specifies a `ClippingPlanesCollection` to clip the terrain. [#5996](https://github.com/AnalyticalGraphicsInc/cesium/pull/5996)
+* Added `Plane.transformPlane` function to apply a transformation to a plane. [#5966](https://github.com/AnalyticalGraphicsInc/cesium/pull/5966)
+* Added `PlaneGeometry`, `PlaneOutlineGeometry`, and `PlaneOutlineGeometryUpdater` classes to render plane primitives. [#5996](https://github.com/AnalyticalGraphicsInc/cesium/pull/5996)
+* Added `PlaneGraphics` class and `plane` property to `Entity`. [#5996](https://github.com/AnalyticalGraphicsInc/cesium/pull/5996)
+
```

Comment:
All these bullets should go under a clipping planes heading. See shadows for example: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md#122---2016-06-01